### PR TITLE
✅ Tests/increase test coverage

### PR DESCRIPTION
**Pull request by Thomas-Smets**

### CODE REVIEW

File: src/test/10_EndToEnd.t.sol [hunk 1]
```diff
@@ -6,274 +6,32 @@
...
-        bayc.transferFrom(tokenCreatorAddress, vaultOwner, 3);
-        mayc.transferFrom(tokenCreatorAddress, vaultOwner, 0);
-        dickButs.transferFrom(tokenCreatorAddress, vaultOwner, 0);
-        interleave.safeTransferFrom(
-            tokenCreatorAddress,
-            vaultOwner,
-            1,
-            100000,
-            "0x0000000000000000000000000000000000000000000000000000000000000000"
+        liquidator = new Liquidator(
```
Comment (j-vp) [inline src/test/10_EndToEnd.t.sol:27]: any reason why this isn't in the general setup?

File: src/test/fixtures/ArcadiaVaultsFixture.f.sol [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+    ArcadiaOracle public oracleEthToUsd;
+    ArcadiaOracle public oracleLinkToUsd;
+    ArcadiaOracle public oracleSnxToEth;
+    ArcadiaOracle public oracleWbaycToEth;
+    ArcadiaOracle public oracleWmaycToUsd;
+    ArcadiaOracle public oracleInterleaveToEth;
+    MainRegistry public mainRegistry;
+    StandardERC20PricingModule public standardERC20PricingModule;
+    FloorERC721PricingModule public floorERC721PricingModule;
+    FloorERC1155PricingModule public floorERC1155PricingModule;
```
Comment (j-vp) [inline src/test/fixtures/ArcadiaVaultsFixture.f.sol:54]: rationale between having these three here and not all (or one)?

File: src/test/13_ProxyUpgrade.t.sol [hunk 3]
```diff
@@ -6,275 +6,56 @@
...
+        address erc20Stored;
+        address erc721Stored;
+        address erc1155Stored;
+        uint256 erc721TokenIds;
+        uint256 erc1155TokenIds;
+        address registry;
+        address trustedProtocol;
+        uint256 life;
+        address owner;
+        VaultInfo vaultVar;
```
Comment (j-vp) [inline src/test/13_ProxyUpgrade.t.sol:42]: todo in a new issue: expand it for vaults with open positions

File: src/test/10_EndToEnd.t.sol (hunk 1 above)
Comment (Thomas-Smets) [reply src/test/10_EndToEnd.t.sol:27]: Liquidator should actually be for Arcadia Lending, so was doubting to include in the ArcadiaVaultsFixture

File: src/test/fixtures/ArcadiaVaultsFixture.f.sol (hunk 2 above)
Comment (Thomas-Smets) [reply src/test/fixtures/ArcadiaVaultsFixture.f.sol:54]: Many test files (Mainregistry, EndToEnd, Gastests) use those 3 and covers the different tokenstandards

File: src/test/8_Vault.t.sol [hunk 4]
```diff
@@ -315,95 +71,33 @@ contract vaultTests is Test {
...
-                baseCurrencyToUsdOracle: address(oracleEthToUsd),
-                baseCurrencyLabel: "ETH",
-                baseCurrencyUnitCorrection: uint64(10 ** (18 - Constants.ethDecimals))
-            }),
-            emptyListUint16,
-            emptyListUint16
-        );
-        vm.stopPrank();
+        vm.prank(vaultOwner);
+        vault_ = new VaultTestExtension();
```
[the remark this answers is not in the document]
Comment (Thomas-Smets) [reply src/test/8_Vault.t.sol:75]: Don't think so, it is a different type (Vault vs VaultTestExtension)

[the remark this answers is not in the document]
Comment (Thomas-Smets) [reply src/test/8_Vault.t.sol:75]: Tested it:
image

**Review by j-vp — APPROVED** [review]
good thing we don't get paid per line!

**Pull request merged**